src/components/QuestionPlay/WithoutOptions.tsx

```diff
@@ -28,6 +53,24 @@ export default function WithoutOptions({
     }, 100);
   }
 
+  function handleSubmitResponse(choice: number[]) {
+    setUserResponse({
+      isConfirmed: true,
+      value: choice,
+    });
+    setTimer(timer + 1);
```

Comment:
We will decrement after 1s not now. Just set isUndoTimerRunning = true and store the ref of userResponse.

Reply:
I'm showing current time in undo button, that is why storing in state
<img width="315" alt="Screenshot 2023-10-20 at 12 23 43 PM" src="https://github.com/rajatkantinandi/quizmaster/assets/14038952/e3c862d3-b5a3-418f-bab2-bf80fd18da6d">

Reply:
OK